api/src/routes/edit.test.ts

```diff
@@ -99,6 +99,65 @@ describe("edit", () => {
         });
     });
 
+    describe("post /object/:pid/datastream/:stream/license", () => {
+        let datastreamManager;
+        let licenseKey;
+        beforeEach(() => {
```

Comment:
Might be a good idea to have an `afterEach` to reset the mocks.

Reply:
Made change.